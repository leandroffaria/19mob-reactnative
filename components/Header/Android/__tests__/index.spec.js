import React from 'react';
import { render } from '@testing-library/react-native';

import HeaderAndroid from '../index';
import Logo from '../../../../assets/mercado-livre.png';

const element = render(<HeaderAndroid logo={ Logo } />);

describe('Header -> Android', () => {
    it('should be image with mercado-livre.png', () => {
        const { getByTestId } = element;

        expect(getByTestId('image').props.source)
            .toEqual({ testUri: '../../../assets/mercado-livre.png' });
    });

    it('should be render snapshot', () => {
        const { container } = element;
        expect(container).toMatchSnapshot();
    });
});