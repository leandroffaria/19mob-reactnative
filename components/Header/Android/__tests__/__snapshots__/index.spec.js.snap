// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header -> Android should be render snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flex": 1,
        "justifyContent": "center",
      }
    }
    testID="container"
  >
    <Image
      source={
        Object {
          "testUri": "../../../assets/mercado-livre.png",
        }
      }
      style={
        Object {
          "height": 33,
          "width": 130,
        }
      }
      testID="image"
    />
  </View>
</View>
`;
